apps/studio/components/interfaces/Connect/Connect.tsx

```diff
@@ -264,6 +266,20 @@ export const Connect = () => {
     }
   }
 
+  useEffect(() => {
+    if (!showConnect) return
+    const hasConnectTabInUrl = typeof router.query.connectTab !== 'undefined'
+    if (!hasConnectTabInUrl && (queryFramework || queryUsing || queryWith)) {
```

Comment:
nit: a little easier to read in my opinion

 
```suggestion
    const noConnectTabInUrl = typeof router.query.connectTab === 'undefined'
    const hasQuery = queryFramework || queryUsing || queryWith
    if (noConnectTabInUrl && hasQuery) {
```